src/main/java/org/folio/rest/impl/OverdueFinePoliciesAPI.java

```diff
@@ -48,6 +55,13 @@ public void postOverdueFinesPolicies(String lang,
     Handler<AsyncResult<Response>> asyncResultHandler,
     Context vertxContext) {
 
+    if (isAnyNegativeQuantity(entity)) {
+      asyncResultHandler.handle(
+        succeededFuture(respond422WithApplicationJson(createError(NEGATIVE_QUANTITY_MESSAGE,
+          HttpStatus.HTTP_UNPROCESSABLE_ENTITY.name()))));
```

Comment:
`createError` accepts error code as 2nd argument, which is not the same thing as HTTP status code

Reply:
sure, but respond422WithApplicationJson() does it